DESCRIPTION

```diff
@@ -37,3 +37,4 @@ Imports:
     grDevices,
     xtable (>= 1.8.4)
```

Comment:
I think we'll probably need to add a `magrittr` dep if we're using the `%>%` in a bunch of places. I'm in favor of that. I think that being more tidy-forward is a good idea. Also will need `tibble` I believe.

I'm surprised the checks are passing without those being included here. Is there a reason for that?

Reply:
Agreed.  I'll  add the pipe and dependency on `tibble` this morning.

The reason that `R CMD CHECK` doesn't complain is because right now the package imports all of this:

```
import(dplyr)
import(ggplot2)
import(httr)
import(methods)
import(utils)
```

Since all of `dplyr` is imported, the pipe is also available.

Importing entire packages, rather than just selected functions, is a bad idea.  We need to fix this.